Makefile

```diff
@@ -90,6 +90,10 @@ nat-libs-sub:
 		$(ENV_SCRIPT) $(MAKE) USE_SYSTEM_NIM=1 nat-libs
 
 deps: | deps-common nat-libs nat-libs-sub
+		echo "Updating nim-secp256k1 path"; \
+		SECP256K1_DIR=$$(readlink -f ./vendor/nim-secp256k1/); \
+		echo $$SECP256K1_DIR > vendor/.nimble/pkgs/nim-secp256k1-#head/nim-secp256k1.nimble-link; \
```

Comment:
Why is this necessary? We don't have such a step in the other projects using nim-secp256k1

Reply:
Without this, when I import secp256k1 it will use `./vendor/nim-waku/vendor/nim-secp256k1` instead of `./vendor/nim-secp256k1`

Reply:
I think we're going to have to reconcile this some other way. It seems to cause a problem when compiling on Windows:
```
D:\a\nim-status\nim-status\nim_status\lib\alias.nim(2, 8) Error: cannot open file: secp256k1
```

Can we reconcile the version of nim-secp256k1 used by us at the top-level and the one in waku's vendor?

Reply:
With the changes in #115 we shouldn't need to do this.